components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -1374,6 +1387,11 @@ impl Animatable for MaxLength {
     }
 }
 
+impl ToAnimatedZero for MaxLength {
+    #[inline]
+    fn to_animated_zero(&self) -> Result<Self, ()> { Err(()) }
```

Comment:
These ones seem... dubious. Mind checking with @hiikezoe or @BorisChiou to see if it's worth to open issues about this? (or do so directly)